keyspace/interceptor/kv_intercepetor.go

```diff
@@ -0,0 +1,25 @@
+package interceptor
```

Comment:
Do we need to introduce `keyspace`? we will attach `resource group name` in request, ref https://github.com/pingcap/kvproto/pull/1029, so maybe no need to introduce `keyspace`.